src/app/shared/report-header/report-header.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { PopoverController } from '@ionic/angular';
+import { PopoverComponent } from '../popover/popover.component';
+
+@Component({
+  selector: 'app-report-header',
+  templateUrl: './report-header.component.html',
+  styleUrls: ['./report-header.component.scss'],
+})
+export class ReportHeaderComponent {
+  @Input() reportType: any;
+  @Input() reportTitle: any;
+  @Output() emitReportType = new EventEmitter<any>();
+  @Output() emitReportAction = new EventEmitter<any>();
+
+  isOpen = false;
+
+  constructor(public popoverController: PopoverController,) {}
+
+  getReportType(type: any) {
+    this.emitReportType.emit(type);
+  }
+
+  presentPopoverforTask(e: Event) {
+    this.isOpen = true;
+  }
+
+  async openPopover(ev: any) {
+    let menu:any =[
+      {
+        title: 'Share',
+        value: 'Share',
+      },
+      {
+        title: 'Download',
+        value: 'Download',
```

Comment:
the values u can keep in lowercase
Ex. {
        title: 'Download',
        value: 'download'}

Reply:
done

Reply:
done